v2/contribute/decisions/session/0019-try-refresh-token-in-optional-session-verification.mdx

```diff
@@ -0,0 +1,31 @@
+---
+id: "0019"
+title: Optional session verification should propagate TRY_REFRESH_TOKEN errors
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Optional session verification should propagate TRY_REFRESH_TOKEN errors
```

Comment:
I made it propagate since the TRY_REFRESH_TOKEN is coming from the validation of the access token, but I'll change it to throw - it makes no difference where the error comes from.